aggsender/validator/validate_db_test.go

```diff
@@ -83,7 +83,7 @@ func TestValidateFullAggsenderDB(t *testing.T) {
 		certificateValidator)
 
 	mockRollupDataQuerier.EXPECT().GetRollupData(mock.Anything).Return(
-		polygonrollupmanager.PolygonRollupManagerRollupDataReturn{}, nil)
+		polygonrollupmanager.PolygonRollupManagerRollupDataReturn{}, nil).Maybe()
```

Comment:
It seems like `.Maybe()` is not required?

```suggestion
		polygonrollupmanager.PolygonRollupManagerRollupDataReturn{}, nil)
```

Reply:
needed otherwise tests are failing